src/dmd/dsymbolsem.d

```diff
@@ -569,6 +569,29 @@ const(char)* getMessage(DeprecatedDeclaration dd)
     return dd.msgstr;
 }
 
+// Returns true if a contact can appear with a function body.
```

Comment:
s/contact/contract/
s/with/without/

Reply:
The function comment header should use DDoc formatting with `Params` and `Returns`.  See https://dlang.org/spec/ddoc.html#standard_srction

Reply:
Not if it's private IMO.

Reply:
I think it's appropriate to use the standard sections but not make it a DDoc comment.